GsaGH/Helpers/GsaAPI/GsaElement.cs

```diff
@@ -0,0 +1,278 @@
+          LoadPanelElelment.Name = value;
+        }
+        else {
+          Element.Name = value;
+        }
+      }
+    }
+
+    public int Property {
+      get {
+        if (IsLoadPanel) {
+          return LoadPanelElelment.Property;
+        }
+        else {
+          return Element.Property;
+        }
+      }
+      set {
+        if (IsLoadPanel) {
+          LoadPanelElelment.Property = value;
+        }
+        else {
+          Element.Property = value;
+        }
+      }
+    }
+
+    public int Group {
+      get {
+        if (IsLoadPanel) {
+          return LoadPanelElelment.Group;
+        }
+        else {
+          return Element.Group;
+        }
+      }
+      set {
+        if (IsLoadPanel) {
+          LoadPanelElelment.Group = value;
+        }
+        else {
+          Element.Group = value;
+        }
+      }
+    }
+
+    public ParentMember ParentMember {
+      get {
+        if (IsLoadPanel) {
+          return LoadPanelElelment.ParentMember;
+        }
+        else {
+          return Element.ParentMember;
+        }
+      }
+      set {
+        if (IsLoadPanel) {
+          LoadPanelElelment.ParentMember = value;
+        }
+        else {
+          Element.ParentMember = value;
+        }
+      }
+    }
+
+    public bool IsDummy {
+      get {
+        if (IsLoadPanel) {
+          return LoadPanelElelment.IsDummy;
+        }
+        else {
+          return Element.IsDummy;
+        }
+      }
+      set {
+        if (IsLoadPanel) {
+          LoadPanelElelment.IsDummy = value;
+        }
+        else {
+          Element.IsDummy = value;
+        }
+      }
+    }
+
+    public Offset Offset {
+      get {
+        if (IsLoadPanel) {
+          throw new ArgumentException("Offset is not applicable for load panels");
+        }
+        else {
+          return Element.Offset;
+        }
+      }
+      set {
+        if (!IsLoadPanel) {
+          Element.Offset = value;
+        }
+        else { throw new ArgumentException("Offset is not applicable for load panels"); }
+      }
+    }
+
+    public double OrientationAngle {
+      get {
+        if (IsLoadPanel) {
+          return LoadPanelElelment.OrientationAngle;
+        }
+        else {
+          return Element.OrientationAngle;
+        }
+      }
+      set {
+        if (IsLoadPanel) {
+          LoadPanelElelment.OrientationAngle = value;
+        }
+        else {
+          Element.OrientationAngle = value;
+        }
+      }
+    }
+
+    public int OrientationNode {
+      get {
+        if (IsLoadPanel) {
+          throw new ArgumentException("Orientation angle is not applicable for load panels");
+        }
+        else {
+          return Element.OrientationNode;
+        }
+      }
+      set {
+        if (!IsLoadPanel) {
+          Element.OrientationNode = value;
+        }
+        else { throw new ArgumentException("Orientation angle is not applicable for load panels"); }
+      }
```

Comment:
The exception should say OrientationNode (not "Orientation angle")

Reply:
@RaviKumarGubbala done